### ISSUE TITLE
Icons customise

### ISSUE THREAD

**Issue by DsPtCh** [osmandapp/OsmAnd#11545]
I used your program for the 2 years. I try to found a way or a plugins for customise my favorite icon.

Like a fish for indicating my fishing place, and different observation I make when I’m in the forest.

I’d like to use my own Icons… is there a way, a format, or something you can do for us ?

I work with your program... you can't imagine how it's good for us. thk's for you work!

**Comment by tradavyvy** [comment]
I would love this feature too

**Comment by dremelts** [comment]
Exactly what I am looking for also.

**Comment by Off-Tracker** [comment]
Is it possible to progress this? OsmAnd has lots of icons but they are tailored for street-drivers, not off-street-walkers. We need things like track branch, track junction, stream crossing, saddle, knoll, cliff; which are not in the OsmAnd set. Other types of users have their own needs. The easiest approach (for users) might be to provide for a custom_icons folder into which we could place a small number (20 is plenty) of icons constructed in any icon editor. according to specs from OsmAnd on icon size, format and naming. For example, Garmin uses a similar approach with 16x16 pixel bmp files.

**Comment by Off-Tracker** [comment]
No, that is just about searching & using built-in POI icons, not adding or using custom ones. Closed issue, but still not possible?

**Comment by dmpr0** [comment]
@Off-Tracker Yes, it's not possible now.

**Comment by Off-Tracker** [comment]
Thanks dmpro. 
I can't find any way to add custom waypoint/favorite icons (ie ones not in the current OsmAnd list). 
In other places it is said not to be possible: https://github.com/osmandapp/OsmAnd/discussions/24198.
Does this mean users must create a custom OsmAnd build? Surely that is beyond most users, and excessive to add a few custom icons.
If it is possible without making app builds, can you please explain how?

**Comment by dmpr0** [comment]
> If it is possible without making app builds, can you please explain how?

As far as I know, no, it's not possible.

**Comment by Off-Tracker** [comment]
Thanks dmpr0. 

It is easy in some"competing" navigation software. The user just needs to create (or find) the icon in a specified size and format (from the days of bitmaps, before scalable vector graphics), and either place it in a specified directory or select it under the waypoint properties dialog. _eg_: https://support.garmin.com/en-US/?faq=gtDjiglxSO5nLNr3BP8m36

It  would be a great feature to include in future versions of OsmAnd.
